composer.json

```diff
@@ -3,9 +3,10 @@
     "description": "PSR-11 Container extension for Behat",
     "type": "library",
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
         "behat/behat": "^3.11.0",
         "psr/container": "^1.1.2",
+        "symfony/config": "^6.0",
```

Comment:
Good catch! This was indeed used in the `PsrContainerExtension`, but never declared! 

https://github.com/Roave/behat-psr11extension/blob/7155882d46c0ad8e70bc2baa01299f178d763bb6/src/PsrContainerExtension.php#L10C5-L10C29

Reply:
Yeah, unit tests in php 8.2 and 8.3 were failing because in QA Checks latest jobs symfony/config 7.0 is upgraded which looks like a BC is introduced:

```
1) RoaveTest\BehatPsrContainer\PsrContainerExtensionTest::testConfiguration
TypeError: Symfony\Component\Config\Definition\Builder\NodeDefinition::end(): Return value must be of type Symfony\Component\Config\Definition\Builder\NodeParentInterface, null returned
```

I required `symfony/config ^6.0` which I think it makes sense because symfony config component is being used in:
https://github.com/Roave/behat-psr11extension/blob/7155882d46c0ad8e70bc2baa01299f178d763bb6/src/PsrContainerExtension.php#L10

I was writing if this fix makes sense but you was faster! :fire: 

Thank you

Reply:
This is fine for now: I think this is a job for `maglnet/composer-require-checker`, if it happens again